### PR TITLE
chore: upgrade Yarn from 3.0.0 to 3.0.2

### PR DESCRIPTION
**Pull request by ahuth**
### Summary:

Upgraded Yarn to the latest version, by running [yarn set version latest](https://yarnpkg.com/cli/set/version).

### Test Plan:

- CI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,5 +84,6 @@
     "*.css": [
       "stylelint --fix"
     ]
-  }
+  },
+  "packageManager": "yarn@3.0.2"
```
Comment (ahuth) [inline package.json:88]: The "set version" command added this.

File: yarn.lock [hunk 2]
```diff
@@ -21912,11 +21912,11 @@ resolve@^2.0.0-next.3:
 
 "typescript@patch:typescript@^4.0.2#~builtin<compat/typescript>, typescript@patch:typescript@^4.3.5#~builtin<compat/typescript>":
   version: 4.3.5
-  resolution: "typescript@patch:typescript@npm%3A4.3.5#~builtin<compat/typescript>::version=4.3.5&hash=d8b4e7"
+  resolution: "typescript@patch:typescript@npm%3A4.3.5#~builtin<compat/typescript>::version=4.3.5&hash=32657b"
   bin:
     tsc: bin/tsc
     tsserver: bin/tsserver
-  checksum: bc2c4fdf0f1557fdafe4ef74848c72ebd9c8c60829568248f869121aea2bb20e16649a252431d0acb185ec118143be22bed73d08f64379557810d82756afedde
+  checksum: 4e2edec3c9b7b9a22ad9cbf9371339d9e13bfe44731f0e2d6a5287f991c3b0e3e0ac0c0ebbf9bf6fc2051e30559c789bfc9cf572cd6b836a0222ae141fac1dba
```
Comment (ahuth) [inline yarn.lock:21919]: Yarn patches TypeScript, which is interesting 🤔. Apparently "they" (whoever maintains Yarn) are trying and failing to get some stuff merged in to TS.

Anyway, updating the Yarn version updated this.

**Review by dierat — APPROVED** [review]

**Comment by dierat** [comment]
Really strange that the `Link` story is showing up as empty now in Percy 😕  (I've had this happen a couple of times recently on Along.)

**Comment by ahuth** [comment]
Thanks!

**Pull request merged**